README.md

```diff
@@ -65,8 +65,8 @@ GCC>=5 is ABI compatible for minor versions. To solve multiple minors, there are
 
 | Version                                                                                       | Arch   |  Status, Life cycle  |
 |-----------------------------------------------------------------------------------------------|--------|------------|
-| - [conanio/clang38: clang 3.8](https://hub.docker.com/r/conanio/clang38/)             | x86_64 |  Supported |
-| - [conanio/clang39-x86: clang 3.9](https://hub.docker.com/r/conanio/clang39-x86/)     | x86    |  Supported |
+| - [conanio/clang38: clang 3.8](https://hub.docker.com/r/conanio/clang38/)             | x86_64 |  Deprecated |
+| - [conanio/clang39-x86: clang 3.9](https://hub.docker.com/r/conanio/clang39-x86/)     | x86    |  Deprecated |
 | - [conanio/clang39: clang 3.9](https://hub.docker.com/r/conanio/clang39/)             | x86_64 |  Supported |
 | - [conanio/clang40-x86: clang 4.0](https://hub.docker.com/r/conanio/clang40-x86)      | x86    |  Supported |
```

Comment:
Let's remove all x86 containers?

Reply:
I would love, but I don't know the consequences. I vote for yes, as CCI and Bincrafters no longer build x86 packages. WDYT?

Reply:
I'd remove them. They will persist in dockerhub like they are now if anyone is using them.